lib/index.js

```diff
@@ -36,6 +36,10 @@ Plotly.register([
 
     require('./scattermapbox'),
 
+    require('./carpet'),
+    require('./scattercarpet'),
+    require('./contourcarpet'),
```

Comment:
Ah, yes. I'm not sure exactly what's best for a default. Easiest for working was just to bundle it. But it's quite a bit of code, generally speaking, so that I feel like it will require some more nuanced per-bundle decisions.